services/order/main.go

```
@@ -0,0 +1,249 @@
+
+func (s *OrderServer) callDownstream(ctx context.Context, method, targetURL string) (map[string]interface{}, error) {
+	req, err := http.NewRequestWithContext(ctx, method, targetURL, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := s.httpClient.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)